# Message format for sending encoder RPM calculations from Arduino to Raspberry Pi.

int16 rpm_fl  # Front-left wheel RPM
int16 rpm_fr  # Front-right wheel RPM
int16 rpm_bl  # Back-left wheel RPM
int16 rpm_br  # Back-right wheel RPM
